textdb/.gitignore

```diff
@@ -1,13 +0,0 @@
-*.iml
```

Comment:
Shuying and I are both using Eclipse instead of Intellij, we need to add a few more lines to gitignore. I ignore the ".gitignore" itself so it's deleted when I commit. I'll add it back and add a few lines we need to it.